src/crimson/os/seastore/onode_manager/staged-fltree/node.cc

```diff
@@ -1828,6 +1829,7 @@ LeafNode::erase(context_t c, const search_position_t& pos, bool get_next)
     }
     return seastar::now().then(
         [c, &pos, this_ref = std::move(this_ref), this, FNAME] () mutable {
+      assert_moveable(this_ref);
```

Comment:
this_ref  is already moved in capture,  does it have meaning to put assert_moveable here?